agents/windows/plugins/mk_oracle.ps1

```diff
@@ -508,6 +508,7 @@ Function sqlcall {
                #####################################################
                # now we ensure that the async SQL Calls have up-to-date SQL outputs, running this job asynchronously...
                #####################################################
+               $ASYNC_PROC_PATH = "$MK_TEMPDIR\async_proc_$sqlsid"
```

Comment:
I guess, this line should also have the txt ending:
`$ASYNC_PROC_PATH = "$MK_TEMPDIR\async_proc_$sqlsid.txt"`

Beside of that, I guess, this is an important fix!

Reply:
valid point, it works either way, but this makes it look better

Reply:
grmpf.. github won't let me edit this PR (or I'm currently not seeing how to) - would you mind adding the .txt in the merge phase? I think you're merging manually anyhow right as this github repo is not your main repo